local-cli/server/runServer.js

```diff
@@ -15,7 +15,7 @@
 require('../../setupBabel')();
 const InspectorProxy = require('./util/inspectorProxy.js');
 const ReactPackager = require('metro-bundler');
-const Terminal = require('metro-bundler/build/lib/TerminalClass');
+const Terminal = require('metro-bundler/build/lib/Terminal');
```

Comment:
How come Flow is complaining here? Doesn't that mean the project is still using the previous version of `metro-bundler` where it is still named `TerminalClass`? In which case we cannot switch it over just yet?